<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>画笔选择</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSolidLine"/>
    <addaction name="actionDashLine"/>
    <addaction name="actionDotLine"/>
    <addaction name="actionDashDotLine"/>
    <addaction name="actionDashDotDotLine"/>
    <addaction name="actionCustomDashLine"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSolidLine">
   <property name="text">
    <string>SolidLine</string>
   </property>
  </action>
  <action name="actionDashLine">
   <property name="text">
    <string>DashLine</string>
   </property>
  </action>
  <action name="actionDotLine">
   <property name="text">
    <string>DotLine</string>
   </property>
  </action>
  <action name="actionDashDotLine">
   <property name="text">
    <string>DashDotLine</string>
   </property>
  </action>
  <action name="actionDashDotDotLine">
   <property name="text">
    <string>DashDotDotLine</string>
   </property>
  </action>
  <action name="actionCustomDashLine">
   <property name="text">
    <string>CustomDashLine</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
